### PR TITLE
Add import questionnaire script

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
This is a new script to be used to import questionnaires from local to dynamoDB. It also adds a switch to the filesystem script because when running this new script it will be run from a different location so needs slightly different paths

### How to review 
- Tests pass
- Script can be run and duplicates the files in the data folder

### CODE REVIEW

**Comment by rmccar** [comment]
[What Authored Schemas to be migrated into Prod](https://trello.com/c/sNjBu3KE/823-what-authored-schemas-to-be-migrated-into-prod)

File: eq-author-api/scripts/import-json-to-dynamo.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
```
Comment (tgandrews) [inline eq-author-api/scripts/import-json-to-dynamo.js]: The filename is dynamo but the script imports to file system?

File: eq-author-api/scripts/import-json-to-dynamo.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const fs = require("fs");
+const jsonPath = process.env.DATA_DIR;
+const { createQuestionnaire } = require("../utils/datastoreFileSystem");
+
+if (!jsonPath) {
+  throw new Error("Set DATA_DIR environment variable");
+}
+
+const importQuestionnaire = async contents => {
+  return createQuestionnaire({ ...contents, id: contents.id + "_copy" });
```
Comment (tgandrews) [inline eq-author-api/scripts/import-json-to-dynamo.js]: Is that a valid questionnaire id? Or will it clash with our UI routing regex?

File: eq-author-api/scripts/import-json-to-dynamo.js (hunk 1 above)
Comment (rmccar) [reply eq-author-api/scripts/import-json-to-dynamo.js]: At the moment it will import to file system for testing but will eventually import to Dynamo

Comment (tgandrews) [reply eq-author-api/scripts/import-json-to-dynamo.js]: Can we not call dynamo yet?

Comment (rmccar) [reply eq-author-api/scripts/import-json-to-dynamo.js]: Yes ive made a change now to allow this

**Pull request merged**